src/app/pages/TakePhoto/TakePhoto.tsx

```diff
@@ -0,0 +1,64 @@
+import styled from 'styled-components';
+import CamDeviceInterface from '../../components/CamDeviceInterface/CamDeviceInterface';
+import NavBarCamera from '../../components/NavBarCamera/NavBarCamera';
+
+export default function TakePhoto(): JSX.Element {
+  function handleTakePhoto() {
+    const width = 1080;
+    const height = 720;
+
+    const video = document.getElementsByTagName('video')[0];
```

Comment:
This would be great as a useRef, but it's fine for now!

Reply:
I'll keep that on my toDo list ;-)